tensorflow/lite/delegates/gpu/gl/kernels/add.cc

```diff
@@ -102,6 +99,28 @@ class Add : public NodeShader {
           /*input=*/IOStructure::AUTO,
           /*output=*/IOStructure::AUTO,
       };
```

Comment:
return early:

return absl::OkStatus();

and when you do that, you don't need to keep a long if else if else sequence.  just break off the else if, and do:

if (hwc_tensor) {

also, do an early return there as well.  you can remove the final else block.

Reply:
The original code does not return early, I did not modify it.